src/FastRouteRouter.php

```diff
@@ -80,15 +109,31 @@ class FastRouteRouter implements RouterInterface
      *     implementation will be used.
      * @param null|callable $dispatcherFactory Callable that will return a
      *     FastRoute dispatcher.
+     * @param array $config Array of custom configuration options.
      */
-    public function __construct(RouteCollector $router = null, callable $dispatcherFactory = null)
-    {
+    public function __construct(
+        RouteCollector $router = null,
+        callable $dispatcherFactory = null,
+        array $config = null
+    ) {
         if (null === $router) {
             $router = $this->createRouter();
         }
 
         $this->router = $router;
         $this->dispatcherCallback = $dispatcherFactory;
+
+        if (isset($config['cache_enabled'])) {
+            $this->cacheEnabled = (bool) $config['cache_enabled'];
+        }
+
+        if (isset($config['cache_file'])) {
```

Comment:
@Ocramius 
I agree with You on all the reasons (Preventing typos, reducing documentation overhead, reducing hidden knowledge, easing refactoring) + ide type-hinting :-).

But i think that every zend component that loads/uses configuration is not using class constants.
Nevertheless, it's not an issue for me to add them. I just tried to use the same "coding" style I saw in other related packages.

kind regards